dbms/src/Common/MemoryTracker.cpp

```diff
@@ -155,6 +171,11 @@ void MemoryTracker::alloc(Int64 size, bool check_memory_limit)
                                   formatReadableSizeWithBinarySuffix(current_limit));
             }
 
+            if (sub_root_of_query_storage_task_mem_trackers)
```

Comment:
how about printing `root_of_non_query` , `sub_root_of_query_storage_task_mem_trackers`, `fetch_pages_mem_tracker` in every case.